app/assets/stylesheets/_vocab-sidebar.scss

```diff
@@ -13,7 +13,7 @@
   z-index: 100;
 
   &:after {
-    background: linear-gradient(transparent, $vocab-bg);
+    background: linear-gradient(transparent, $vocab-bg 2.5rem, $vocab-bg);
```

Comment:
Is this line missing a comma? Or does the variable and the measurement combine somehow here?

Reply:
After this, the height is set to `3rem`, so this measurement makes sure there is a guaranteed `.5rem` of completely blanked out background color. It was hard to see the scroll thing at smaller heights without this. Just a small tweak to make it more obvious. The padding at the bottom is a full rem anyway, so this doesn't get seen once you scroll at the way to the bottom still.